addons/im_livechat/static/tests/embed/livechat_session_tests.js

```diff
@@ -57,6 +57,7 @@ QUnit.test("Seen message is saved on the session", async (assert) => {
     await insertText(".o-mail-Composer-input", "Hello World!");
     triggerHotkey("Enter");
     await contains(".o-mail-Message", { count: 2 });
+    await nextTick(); // wait for message seen
```

Comment:
should check with contains for the seen indicator instead

Reply:
There's no UI clue that this is changed, and the next assertion inspect some internal data...